### ISSUE TITLE
Signposts have accessibility issues

### ISSUE THREAD

**Issue by kglickman** [vmware/clarity#1274]
**Select one ...**  (check one with "x")
```
[x ] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

- When tabbing to a signpost icon, the icon should get a focus outline

### Actual behavior
- The icon has the focus, but there is no visible indication of such.
- I am also getting a message about the icon in an internal accessibility testing tool - Minimum contrast of 2.81 with its background is not sufficient to meet WCAG AA requirements for text of size 12px and weight of 500.

### Reproduction of behavior
This can be reproduced on the Clarity web page https://vmware.github.io/clarity/documentation/signposts in the section titled "Signpost positions"

### Environment details

* **Angular version:** 4.1.3

* **Clarity version:** 0.9.13

* **OS and version:** Windows 7

* **Browser:** [all | Chrome 60 | IE 11

**Comment by mathisscott** [comment]
1) The signpost icon should have focus indicated with the outline as well as the icon turning action blue.

2) To handle WCAG AA compliance, the signpost icon should be #737373 when not highlighted/focused and #0065AB when hovered/focused.

**Comment by mathisscott** [comment]
NOTE: this may require some images in the documentation to change

**Comment by reddolan** [comment]
I am ok with both of @mathisscott's recommendations. From the naked eye, the changes aren't perceptible. 

This does bring up a greater challenge which is about our default icon color being #747474. It is not accessible on a #fafafa background. We might want to change our default icon color to #737373 so as to not run into any more accessibility issues. This unfortunately is not part of our color palette cc: @lil-kim
